### PR TITLE
Feature docker readme

### PR DESCRIPTION
**Pull request by mosheBelostotsky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,14 +4,39 @@ Hubot middleware and scripts for enterprise
...
+    -e 'NPM_INTEGRATIONS=<integration to install from npm>'\
+    -e "http_proxy=${http_proxy}" \
+    -e "https_proxy=${https_proxy}" \
+    -e "no_proxy=${no_proxy}" \
+    -e "ADAPTER=slack" \
+	-e "SLACK_APP_TOKEN=xxxxxxxxx"
+	-e "HUBOT_SLACK_TOKEN=xxxxxxxxxxx"
+  chatopshpe/hubot-enterprise
+  ```
+  - **integrations_on_disk**: hubot scripts npms located on local disc, can be either npm project folder or number npm folders prefixed with `hubot-` prefix
```
Comment (rqc) [inline README.md]: The wording here is not very clear. Could you revise?

Comment (rqc) [inline README.md]: add example ...

File: README.md [hunk 2]
```diff
@@ -4,14 +4,39 @@ Hubot middleware and scripts for enterprise
...
+
+  - **NPM_INTEGRATIONS**: list of hubot scripts to be installed using npm
```
Comment (rqc) [inline README.md]: add example.

File: README.md (hunk 1 above)
Comment (rqc) [inline README.md]: add period `.` at the end of sentence.

File: README.md (hunk 2 above)
Comment (rqc) [inline README.md]: add period `.`

File: README.md [hunk 3]
```diff
@@ -4,14 +4,39 @@ Hubot middleware and scripts for enterprise
...
+  - **ADAPTER**: default is slack, here can specify any other adapter to be installed via npm and used by hubot
```
Comment (rqc) [inline README.md]: add period

Comment (rqc) [inline README.md]: sample values...

Comment (mosheBelostotsky) [inline README.md]: @rqc updated.

File: README.md (hunk 2 above)
Comment (mosheBelostotsky) [inline README.md]: @rqc, updated.

File: README.md (hunk 1 above)
Comment (mosheBelostotsky) [inline README.md]: @rqc, updated

**Comment by mosheBelostotsky** [comment]
@rqc @edagan please read latest readme.
Added TOC, removed many stuff from README and referenced to wiki as @edagan requirements

File: README.md [hunk 4]
```diff
@@ -2,45 +2,100 @@
 
 Hubot middleware and scripts for enterprise
 
+This hubot plugins package created to allow quickly enable multiplatform hubot plugins development and deployment with built in 
+security flows, help mechanism, standartized user experience in terms of syntax, message templating and collaboration platforms management API adapters.
+
+For more information, please reference our [wiki](https://github.com/eedevops/hubot-enterprise/wiki)
+
+## Quick links for README
```
Comment (rqc) [inline README.md]: having quick links is a good idea, @edagan what do you think?

File: README.md [hunk 5]
```diff
@@ -2,45 +2,100 @@
...
+- [Developing hubot integration with hubot-enterprise support](#developing-hubot-integration-with-hubot-enterprise-support)
+  - [Creating new integration](https://github.com/eedevops/hubot-enterprise/wiki/bootstrap-integration)
+  - [Writing integration](https://github.com/eedevops/hubot-enterprise/wiki/api)
+  - [Testing your integration](https://github.com/eedevops/hubot-enterprise/wiki/testing)
+  - [Using jenkins pipeline docker image](https://github.com/eedevops/hubot-enterprise/wiki/jenkins)
+- [DISCLAIMER](#disclaimer)
+
 ## Installation
 
+The package can quickly be deployed and installed using docker with the following prerequisites:
```
Comment (rqc) [inline README.md]: @mosheBelostotsky I would even remove all the `Installation` docs as per @edagan suggestion.

**Comment by mosheBelostotsky** [comment]
@rqc @edagan, Done. please review again.

**Comment by mosheBelostotsky** [comment]
@rqc done.

File: README.md [hunk 6]
```diff
@@ -1,109 +1,25 @@
...
+
+For more information, please reference our [wiki](https://github.com/eedevops/hubot-enterprise/wiki)
+
+## Quick links
+- [Installation](https://github.com/eedevops/hubot-enterprise/wiki/bootstrap)
+  - [Colaboration tool](https://github.com/eedevops/hubot-enterprise/wiki/bootstrap#1-collaboration-tool-select-one-and-set-up-integration-with-hubot)
+  - [Via docker](https://github.com/eedevops/hubot-enterprise/wiki/bootstrap#docker)
+  - [Other installation methods](https://github.com/eedevops/hubot-enterprise/wiki/bootstrap#other-installation-methods)
+- [Built in commands](https://github.com/eedevops/hubot-enterprise/wiki/hubot-admin)
+- [Developing hubot integration with hubot-enterprise support](#developing-hubot-integration-with-hubot-enterprise-support)
```
Comment (rqc) [inline README.md]: This link doesn't work. Is it a link to the wiki? If not and if it just a title for that section of links name it `Developing integrations with hubot-enterprise`.

**Comment by rqc** [comment]
@mosheBelostotsky I have one more request. Could you squash all commits to one (or at least a handful?). I don't think we need all of those incremental tweaks in our history. Once that's done, please merge...

**Pull request merged**